e2e/ansible/playbooks/hyperconverged/test-k8s-memleak/test-mem.py

```diff
@@ -0,0 +1,31 @@
+#------------------------------------------------------------------------------
+#!/usr/bin/env python
+#description     :Test to verify the memory consumed with sample workloads. 
+#==============================================================================
+
+from __future__ import division
+import subprocess
+import time, os
+list = []
+n = 10
+count = 0
+while count < n:
+    count = count + 1
+    #Obtaining memory consumed using free command from the node where the controller
+    #is scheduled.
+    cmd_used_mem = "sudo free | awk ' /'Mem'/ {print $3}'"
```

Comment:
You are grabbing system memory not process memory.

Reply:
@gila I will do this task as enhancement as part of next sprint and track under https://github.com/openebs/openebs/issues/1350

Reply:
@gila Now modified the code to grab process memory.